setup.cfg

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --reuse-db --instafail --showlocals
+addopts =-vvs --reuse-db --instafail --showlocals --tb=short
```

Comment:
I found that the combination -vvs and short tracebacks lead to a pretty good output but I'm open to suggestions.